modules/CNN_constructor.py

```diff
@@ -0,0 +1,128 @@
+""" Module contain a few constructors for CNN.
+
+"""
+
+import torch
+from torch import nn
+
+from modules import CNN_blocks_rev
```

Comment:
There is no module called `CNN_blocks_rev`

Reply:
Why don't you just 
`from ... import ResNetNormalBlock, ResNetBottleneckBlock`?